huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -147,261 +230,222 @@ func buildMaintainEndTime(maintainStart string) (string, error) {
 	regex := regexp.MustCompile("^(02|06|10|14|18|22):00:00$")
 	isMatched := regex.MatchString(maintainStart)
 	if !isMatched {
-		return "", fmtp.Errorf("The start-time format of maintenance window is not 'xx:00:00' or " +
+		return "", fmt.Errorf("the start-time format of maintenance window is not 'xx:00:00' or " +
```

Comment:
this value has been validated in the schema, No need to check again

Reply:
OFC, these codes have been removed.